components/search_bar/search_bar.jsx

```diff
@@ -252,11 +254,11 @@ export default class SearchBar extends React.Component {
                     className='search-form__container'
                 >
                     <form
-                        role='application'
```

Comment:
@asaadmahmood Are there any ramifications to removing this?

Reply:
I think one of our major customers was having an issue with the cursor mode changing from section to section, so I would say we keep it.
Plus, there is an autocomplete there, which would stop working if we make this change (in terms of readout etc).